### PR TITLE
Add Galician language

### PR DESCRIPTION
**Pull request by frandieguez**
Please consider adding these changes to support the Galician language in Dialect

### CODE REVIEW

File: gl.po [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+msgstr "Copyright © 2020–2021 Os autores de Dialecto"
+
+#: data/resources/about.ui:14
+msgid "GitHub page"
+msgstr "Páxina de GitHub"
+
+#. Replace me with your names.
+#: data/resources/about.ui:19
+msgid "translator-credits"
+msgstr "Rafael Mardojai CM"
```
Comment (fushinari) [inline gl.po:91]: Please replace this with your name :)

Comment (frandieguez) [reply gl.po]: done

File: LINGUAS [hunk 2]
```diff
@@ -1,27 +1,26 @@
+ca
 cs
 de
+eo
```
Comment (fushinari) [inline LINGUAS:4]: Is it possible to not make any changes to this file other than adding your language?

Comment (frandieguez) [reply LINGUAS:4]: I'm just sorting language ids alphabetically.

Comment (fushinari) [reply LINGUAS:4]: A few languages are missing though, immediately noticeable is the missing "ko" and "oc" for example.

Comment (frandieguez) [reply LINGUAS:4]: You are right, I fixed it in my master branch

**Pull request merged**